### PR TITLE
Added initial draft for workstream 2's spec

### PR DESCRIPTION
**Pull request by ssuffian**
This is an initial draft of the workstream 2 spec (data types), put into the specification folder using the template.

### CODE REVIEW

File: specifications/cdsc-wg2-02.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+* <b>Market</b>. A market is a collection of nodes that are connected to each other through a single market operator. A market can for example be a single balancing area, or a collection of balancing areas. A <b>Market</b> SHOULD represent Granularity Levels 3 defined by `CDSC-WG2-01`
+* <b>Unit</b>. A unit is a single physical entity. A unit can for example be a single power plant, or a single generation unit within that plant. A <b>Unit</b> SHOULD represent Granularity Levels 4,5 defined by `CDSC-WG2-01`.
+
+### System<a id="entities--system" href="#entities--system" class="permalink">🔗</a>
+
+| Data Type                           | Data definition                                                          | Unit / Data Format | Temporal Granularity          | Secondary Information                        | Locational Granularity | Time horizon        | Latency   | Maximum Latency | Requirement level |
```
Comment (halliecramer1) [inline specifications/cdsc-wg2-02.md:73]: should terminology for "Locational Granularity" align with CDSC-WG2-01? i.e. reference levels 1-2?

File: specifications/cdsc-wg2-02.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+| Exports                             | Exports by interconnect                                                  | MW (Float)         | Market (hourly or sub hourly) | Locational Granularity                       | Balancing Region       | Real-time           | Real-time | Day +7          | Must-have         |
+| Renewable generation curtailment    | Curtailment of renewable generation                                      | MW (Float)         | Market (hourly or sub hourly) | Locational Granularity, fuel type (optional) | Balancing Region       | Real-time           | Real-time | Day +7          | Should-have       |
+| Generation by fuel type             | The full breakdown of generation per fuel type (e.g. wind, solar)        | MW (Float)         | Market (hourly or sub hourly) | Fuel Type List                               | Balancing Region       | Real-time           | Real-time | Day +7          | Must-have         |
```
Comment (halliecramer1) [inline specifications/cdsc-wg2-02.md:81]: tradeoff of having this on it's own vs adding "Fuel Type List" to secondary information as part of the Generation row?

File: specifications/cdsc-wg2-02.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+| Demand Forecast                     | Forecast of system demand over the next day                              | MW (Float)         | Market (hourly or sub hourly) | Locational Granularity                       | Balancing Region       | Short-term forecast | Day-ahead | N/A             | Should-have       |
+| Generation Forecast                 | Forecast of system generation by fuel type over the next day             | MW (Float)         | Market (hourly or sub hourly) | Fuel Type List, generation technology        | Balancing Region       | Short-term forecast | Day-ahead | N/A             | Should-have       |
```
Comment (halliecramer1) [inline specifications/cdsc-wg2-02.md:83]: where we say "Fuel Type List" do we actually mean Fuel Type + Generation Technology?

File: specifications/cdsc-wg2-02.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+|--------------------------------------------------------------|------------------------------------------------------------------------------------|----------------------|-------------------------------|------------------------|------------------------|--------------|-----------|-----------------|-------------------|
+| Real-Time market price/ Locational Marginal Prices/ Balancing | LMP set by system operator at each node on the grid (assuming nodal market design) | Currency/MWh (Float) | Market (hourly or sub hourly) | Locational Granularity | Zonal / nodal          | Real-time    | Real-time | N/A             | Should-have       |
+| Locational Marginal Congestion Prices                        | The impact of congestion on LMP                                                    | Currency/MWh (Float) | Market (hourly or sub hourly) | Locational Granularity | Zonal / nodal          | Historical   | Day +?    | N/A             | Should-have       |
```
Comment (halliecramer1) [inline specifications/cdsc-wg2-02.md:94]: Day +7

File: specifications/cdsc-wg2-02.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+
+### Market<a id="entities--market" href="#entities--market" class="permalink">🔗</a>
+
+| Data Type                                                    | Data definition                                                                    | Unit / Data Format   | Temporal Granularity          | Secondary Information  | Locational Granularity | Time horizon | Latency   | Maximum Latency | Requirement level |
+|--------------------------------------------------------------|------------------------------------------------------------------------------------|----------------------|-------------------------------|------------------------|------------------------|--------------|-----------|-----------------|-------------------|
+| Real-Time market price/ Locational Marginal Prices/ Balancing | LMP set by system operator at each node on the grid (assuming nodal market design) | Currency/MWh (Float) | Market (hourly or sub hourly) | Locational Granularity | Zonal / nodal          | Real-time    | Real-time | N/A             | Should-have       |
```
Comment (halliecramer1) [inline specifications/cdsc-wg2-02.md:93]: suggest choosing one name and then using the definition to make clear what it means (cross-geographically)

File: specifications/cdsc-wg2-02.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+| Heat Rates Curves                  | The amount of energy used by an electrical generator/power plant to generate one kWh of electricity at different generation levels | J/kWh (Float)                            | Monthly                       | Unit ID                 | unit                   | Historical   | Annual  | Year + 1        | May-have          |
+| Unit fuel types                    | Fuel consumed by each unit                                                                                                         | List of types (List[String])             | Annual                        | Unit ID                 | unit                   | Historical   | Annual  | Year + 1        | Must-have         |
+| Unit Location                      |                                                                                                                                    | Lat,Long (Cf Google Maps) (Tuple[Float]) | Annual                        | Unit ID                 | unit                   | Historical   | Annual  | Year + 1        | Should-have       |
```
Comment (halliecramer1) [inline specifications/cdsc-wg2-02.md:107]: add missing definitions for completeness

**Review by halliecramer1 — COMMENTED** [review]
Overall looks good. Made suggestions to help with clarity.

File: specifications/cdsc-wg2-02.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+  * [3.3 Unit](#entities--unit)
+  * [3.4 Other Entities](#entities--other-entities)  
+
+## Foreword <a id="foreword" href="#foreword" class="permalink">🔗</a>
+
+Attention is drawn to the possibility that some of the elements of this document may be the subject of patent rights. No party shall not be held responsible for identifying any or all such patent rights.
+
+Any trade name used in this document is information given for the convenience of users and does not constitute an endorsement.
+
+This document was prepared by [Insert name of group].
```
Comment (pierresegonne) [inline specifications/cdsc-wg2-02.md:30]:
```suggestion
This document was prepared by the 2nd working group of carbon data specification consortium.
```

File: specifications/cdsc-wg2-02.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+
+This second/third/… edition cancels and replaces the first/second/… edition (#####:####), which has been technically revised.
+The main changes compared to the previous edition are as follows:
+—	xxx xxxxxxx xxx xxxx
```
Comment (pierresegonne) [inline specifications/cdsc-wg2-02.md:34]:
```suggestion
```

**Pull request closed without merging**